zhaquirks/philips/__init__.py

```diff
@@ -19,3 +59,57 @@ class PhilipsOnOffCluster(CustomCluster, OnOff):
 
     attributes = OnOff.attributes.copy()
     attributes.update({0x4003: ("power_on_state", PowerOnState)})
+
+
+class PhilipsBasicCluster(CustomCluster, Basic):
+    """Philips Basic cluster."""
+
+    attributes = Basic.attributes.copy()
+    attributes.update({0x0031: ("philips", t.bitmap16)})
```

Comment:
@dmulcahey where this is coming from? isn't this attr supposed to be on `0x0406` Occupancy cluster id?

Reply:
https://github.com/Koenkk/zigbee-herdsman-converters/blob/69befbcce5d744cf5f81e1cdf6f59cd5bcb5917e/devices.js#L931

Reply:
It makes the notification on the manufacturer cluster work from what I could see. It’s like a mode switch similar to what the Opple and Lightify x4 remotes do

Reply:
ah, ok. Just didn't see this in https://github.com/dresden-elektronik/deconz-rest-plugin/wiki/Philips-Hue-sniffing#hue-motion-sensor and thought it was the same attribute as sensitivity on occupancy cluster